man/adoc/bpftrace.adoc

```diff
@@ -188,6 +188,16 @@ Print bpftrace version information.
 Enable verbose messages.
 For more details see the <<Verbose Output>> section.
 
+=== Program Options
+
+You can also pass custom options to a bpftrace program/script itself via positional or named parameters.
+All positional parameters need to come before any named parameters e.g.
```

Comment:
I noticed this while reading the parsing code, and have a long comment/proposal below. We can discuss there, I just wanted to note that this isn't really how positional parameters normally work, and therefore we would be introducing a rather sharp edge right away.